config/config.go

```diff
@@ -10,23 +10,24 @@ const KafkaTLSProtocolFlag = "TLS"
 
 // Config represents service configuration for dp-search-data-finder
 type Config struct {
+	APIRouterURL               string        `envconfig:"API_ROUTER_URL"`
 	BindAddr                   string        `envconfig:"BIND_ADDR"`
 	ContentUpdatedTopicFlag    bool          `envconfig:"CONTENT_UPDATED_TOPIC_FLAG"`
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	KafkaConfig                KafkaConfig
+	ServiceAuthToken           string        `envconfig:"SERVICE_AUTH_TOKEN"   json:"-"`
 	ZebedeeClientTimeout       time.Duration `envconfig:"ZEBEDEE_CLIENT_TIMEOUT"`
-	ZebedeeURL                 string        `envconfig:"ZEBEDEE_URL"`
 }
 
 // KafkaConfig contains the config required to connect to Kafka
 type KafkaConfig struct {
 	Brokers               []string `envconfig:"KAFKA_ADDR"`
 	ContentUpdatedTopic   string   `envconfig:"KAFKA_CONTENT_UPDATED_TOPIC"`
+	ConsumerGroup         string   `envconfig:"KAFKA_CONSUMER_GROUP"`
```

Comment:
Have we updated the secrets?

To prevent application failure, you will want both `KAFKA_CONSUMER_GROUP` set and `KAFKA_REINDEX_REQUESTED_GROUP` in secrets, then once these changes have deployed, remove `KAFKA_REINDEX_REQUESTED_GROUP` from secrets

Reply:
If we merge the secrets first and then merge the app afterward, the app should take the new secrets and function properly

Reply:
Secrets - https://github.com/ONSdigital/dp-configs/pull/1043